### ISSUE TITLE
Event code updated in Hubhop is shown in MF Connector, but is shown as custom code, not as preset

### ISSUE THREAD

**Issue by JaimeLeon2** [MobiFlight/MobiFlight-Connector#718]
**Describe the bug**
After updating the code in one event for CJ4 ENG RUNSTOP button in Hubhop, updated the events in MF Connector, the code is displayed in MF Connector correctly, but is not flagged as a preset but rather as custom code.  

**Steps To Reproduce**
1. Update event in Hubhop and save the modified event.
2. Update events in MF Connector from Hubhop
3. Use UI to select the modified event.  Note that the preset is shown and the code is put correctly in the code box.

![image](https://user-images.githubusercontent.com/86563553/155896025-41e1419b-c3bb-4f62-a9aa-aabc34a1a7a6.png)

5. Press Ok and save the config.
6. Reenter the same config.  Note the preset information is not displayed.  The code box has the code, but it is shown as a customized event, not as a preset.

![image](https://user-images.githubusercontent.com/86563553/155896079-d6b8c080-c064-4c93-b347-7ee62a8b5c47.png)

**Expected behavior**
Event should be shown as a preset.

**Actual behavior**
Event is shown as a customized event.

**Details**
 - MobiFlight Version: 9.2.0.5
 - FlightSim: irrelevant
 - AddOn: irrelevant

**Additional context**
none

**Comment by MobiFlight-Admin** [comment]
I know why it happens. It has a :1 which I convert into a :index because I assumed that the preset will not come with the specific :1 (inspired by the COM radio preset)

But I guess I should check without and with the :index replacement to see if i can find a config.

**Comment by JaimeLeon2** [comment]
Yes, I think a generic event would have :index, but a specific airplane event would most likely have a pre-defined index.
